utils/seed_asset.py

```diff
@@ -22,6 +22,14 @@
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        prog="seed_asset",
+        description="""
+            The program for importing of assets
+        """
+    )
+    args = parser.parse_args()
+
```

Comment:
@JaphethCA `if not show the message and continue with execution.` - is this possible?

Reply:
Am sorry, but I don't understand what you mean. Can you please give more details.

Reply:
currently the `-h` argument shows the help text and exits.. without the `-h` it goes straight to execution (asks user to enter the csv file name) without displaying the help text. is it possible to show the help text even without the `-h`?

Reply:
Sure, but how will user run the command to input csv file or link if running the script without arguments return a help text?